src/intl.js

```diff
@@ -22,6 +22,10 @@ const intlStrings = {
         'You currently do not have access to proxy groups: {groups}',
     'inputSchema.validation.customProxyInvalid':
         'Proxy URL "{invalidUrl}" has invalid format, it must be http://[username[:password]]@hostname:port.',
+    'inputSchema.validation.apifyProxyCountryInvalid':
+        'Country code "{invalidCountry}" is invalid.',
```

Comment:
Let's metion in error that `ISO 3166-1 alpha-2 codes` are supported  https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2